util/build_configs/setenv-example-3.bash

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env bash
+
+# Example setenv script to build a customized Chapel Cray module
```

Comment:
I think these examples should include a human-readable description in comments or README of what they actually do configuration-wise.

Reply:
Yes